lib/msal-common/test/client/AuthorizationCodeClient.spec.ts

```diff
@@ -154,7 +154,7 @@ describe("AuthorizationCodeClient unit tests", () => {
             const client = new AuthorizationCodeClient(config);
             const testAccount = TEST_ACCOUNT_INFO;
             // @ts-ignore
-            const testTokenClaims: Required<Omit<TokenClaims, "home_oid"|"upn"|"cloud_instance_host_name"|"cnf"|"emails"|"login_hint">> = {
```

Comment:
This is a great example of why you should do major cleanup/refactoring in a separate PR, GitHub decided not to render this diff by default as it's too large for this file.